client/src/ingredients/specifications.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+export function Specifications({ specifications }) {
```

Comment:
So, sometimes it's a string and sometimes it's an array? 
If so, I'd be more comfortable that you check it more carefully. 
For example:
```javascript
if (typeof specifications === 'string') {
  if (specifications === 'non-standard') {
     return <> <h2>NOn-standard feature</h2> ... </>
  } else {
     throw new Error(`specifications not recognized (${specifications})`);
  }
} else {
   ... as is ...
```